lib/dashboard/DefaultDashboardFactory.ts

```diff
@@ -58,22 +58,30 @@ export interface MonitoringDashboardsProps {
   /**
    * Flag indicating whether the default dashboard should be created.
    * This is independent on other create dashboard flags.
+   *
+   * @default true
    */
-  readonly createDashboard: boolean;
+  readonly createDashboard?: boolean;
   /**
    * Flag indicating whether the summary dashboard should be created.
    * This is independent on other create dashboard flags.
+   *
+   * @default true
    */
   readonly createSummaryDashboard: boolean;
   /**
    * Flag indicating whether the alarm dashboard should be created.
    * This is independent on other create dashboard flags.
+   *
+   * @default true
    */
   readonly createAlarmDashboard: boolean;
   /**
    * Dashboard rendering preference.
+   *
+   * @default DashboardRenderingPreference.INTERACTIVE_ONLY
    */
-  readonly renderingPreference: DashboardRenderingPreference;
+  readonly renderingPreference?: DashboardRenderingPreference;
 }
 
 export class DefaultDashboardFactory
```

Comment:
Do you think there is a way to have `DefaultDashboardFactory` as the default value for `dashboardFactory`? We could derive the `dashboardNamePrefix` somehow.

Reply:
Yeah, might be a good next logical step here.

Reply:
Done in the next rev.